From 02aecd7eafbb43be295d72c6d216ac22b3ba972a Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:39:29 -0400
Subject: [ethernet] i40e/i40evf: Bump i40e to 0.4.17 and i40evf to 0.9.36

Message-id: <1409658042-22689-209-git-send-email-sassmann@redhat.com>
Patchwork-id: 90402
O-Subject: [RHEL7.1 PATCH 208/281] i40e/i40evf: Bump i40e to 0.4.17 and i40evf to 0.9.36
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Bump versions.

Change-ID: I47fc3433240800cd823ff512f3015822277b0d20
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 25941f94ba2dbb0b8e451a917aa12a547994c738)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index b1f7896..10b4028 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -39,7 +39,7 @@ static const char i40e_driver_string[] =
 
 #define DRV_VERSION_MAJOR 0
 #define DRV_VERSION_MINOR 4
-#define DRV_VERSION_BUILD 13
+#define DRV_VERSION_BUILD 17
 #define DRV_VERSION __stringify(DRV_VERSION_MAJOR) "." \
       __stringify(DRV_VERSION_MINOR) "." \
       __stringify(DRV_VERSION_BUILD)    DRV_KERN
-- 
1.7.1